packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -108,40 +94,25 @@ const BIContractActivationWindow = ({
           variant="text"
           color="primary"
           disabled={!extraParams}
-          onClick={() => setWindowVisible(true)}
+          onClick={() => {
```

Comment:
```suggestion
          onClick={async() => {
          try {
           const openedOAuthWindow = await openOAuthWindow()
           afterOAuthWindow()
           catch(err => {
                logger.error('openOAuthWindow error', err)
              })
           
```

Reply:
I supposed it was a bad practice to return a promise to an event handler but could not find anything about it now. Ok let's do this